doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -259,8 +263,7 @@ pass testing for ``start_test -performance``).
 
 .. _Creating a Simple Test: https://github.com/chapel-lang/chapel/blob/main/doc/rst/developer/bestPractices/TestSystem.rst#creating-a-simple-test
 
-
-.. _Contributing changes
+.. _Contributing changes:
 
 Contributing changes
 --------------------
```

Comment:
The "how to push" link below is broken

Reply:
As is the "how to open a PR" link in the "Submit Pull request" section and the two in the "After Merging" section

Reply:
And the "DCO" link in the "Before Merging" section